openshift-api-for-data-protection-operator/base/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: openshift-adp
+  name: openshift-adp
+spec:
+  finalizers:
```

Comment:
Finalizers are also not recommended to include.

k8s will add the necessary finalizers automatically.

Reply:
Thanks, will keep that in mind :)